index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="Product Landing Page">
+    <title>Product Landing Page for Credit and Saving company</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body id="body">
+ <div id="page-wrapper">
+   <header id="header"> 
+     <div class="logo">
+        <img id="header-img" src="https://media.istockphoto.com/vectors/money-saving-logo-vector-id875032110?k=6&m=875032110&s=170667a&w=0&h=VoZEUSV_qM32ScEhZ2ofojMSNshqlQ8XGOMFmGv71Dc=" alt="company logo">
+     </div>
+     <nav id="nav-bar">
+        <ul>
+          <li>
+            <a class="nav-link" href="#saving products">Saving products</a>
+          </li> 
+          
+          <li>
+            <a class="nav-link" href="#Loan Products">Loan Products</a>
+          </li> 
+          
+          <li>
+            <a class="nav-link" href="#About us">About us</a>
+          </li> 
+
+        </ul>
+      </nav>
+    </header>
+    
+    <div class="container"></div> 
```

Comment:
This container doesn't have anything in it,  so you can delete it.

Reply:
- Kindly, phrase the comments politely, as suggestions instead of orders. For example:
        This is a polite suggestion: **"Kindly remove....."** 👌
- Kindly, always includes an explanation, a "why", for each requested change 👌
- [Optional] Great job for adding the comment politely and with reasons 👍,  I think it would be better if you use a couple of emojis and/or gifs in the review, they make the code review more human-friendly 👽🤖.